internal/cli/iam/projects/list.go

```diff
@@ -76,6 +79,21 @@ func ListBuilder() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return opts.Run()
 		},
+		PostRun: func(cmd *cobra.Command, args []string) {
+			now := time.Now()
+			cmdPath := cmd.CommandPath()
+			command := strings.ReplaceAll(cmdPath, " ", "-")
+			var properties = map[string]string{
+				"command": command,
+			}
+			var event = telemetry.Event{
+				Timestamp:  now.Format(time.RFC3339Nano),
+				Source:     "atlascli",
```

Comment:
```suggestion
				Source:     config.Bin(),
```

Reply:
There isn't a "Bin" function - there is a "BinName" function, but it returns just "atlas", whereas the "ToolName" variable contains "atlascli", so I'll use that for both the Source and Event fields:

```
  Source: config.ToolName,
  Name:   config.ToolName + "-event",
```